### ISSUE TITLE
Is this mod good for bot prevention?

### ISSUE THREAD

**Issue by Dani3I** [Mrbysco/Captcha#2]
### General Info

- [ ] I would like to propose new content
- [ ] I would like to propose changes
- [X] I would like to propose another thing

### Suggestion Description

Hi, so i am planning to make fabric server and I am looking for mod that would show captcha to prevent bots. Basically looking for something that would do the same things that "[Captcha](https://www.spigotmc.org/resources/captcha.75891/)" plugin for spigot created by "Kyllian Warmerdam" ("InstantlyMoist"). It looks like this project was made for joke and not for use-case that I am looking for but I am gonna as anyway:
- does this mod works good in multiplayer?
- does this mod is secure (I mean if the captchas are possible to bypass by some bots etc.)?
- i didn't downloaded mod yet so maybe it is explained elsewhere, but can i disable "captchaCooldown", so it only asks for captcha for new players that are joining to my server?

In other words, should I use this mod or try using mentioned before plugin with "[Cardboard](https://cardboardpowered.org/)" mod for fabric that tries to make standard plugins work with fabric.

**Comment by Dani3I** [comment]
I am dumb, sorry. I googled fabric Captcha, but didn't checked if this mod is for fabric or for forge. Looks like it if for forge and not for fabric.

**Comment by xllifi** [comment]
@Dani3I now for fabric too
